docs/collections/keyvalue/index.md

```diff
@@ -8,6 +8,33 @@ import TabItem from '@theme/TabItem';
 
 A key-value collection stores data in the form of a dictionary. This requires a key for each record and a corresponding value that contains the data. Each document in the database must have a unique key, similar to the concept of a primary key. Key-value databases are known for being quick and storage efficient resulting in faster CRUD (Create, Read, Update, Delete) operations.
 
-In GDN, each document stored in a collection contains a `_key`, and the rest of the document is its _value_. The only key-value operations available are key lookups (single and batch) and key-value pair insertions and updates. If no sharding attribute is specified, `_key` is used for sharding the data instead.
+## Key-Value Store Contents
+
+In GDN, each key-value pair (document) stored in a collection contains a `_key`, and the rest of the document is its _value_. The only key-value operations available are key lookups (single and batch) and key-value pair insertions and updates. If no sharding attribute is specified, then `_key` is used for sharding the data instead.
 
 Key-value collections are always global. You can specify time_to_live (TTL) during creation.
+
+## Group ID
+
+When you create a key-value store collection, you can select the **Group** option to enable the **GROUP ID** field. This allows you to organize key-value pairs in groups. This option has several effects:
+
+- The Group ID field is automatically indexed, which makes querying based on the ID faster.
+- You can perform queries and truncate based on the group ID value.
+
+Here are some example queries:
+
+```sql
+FOR d IN <groupID> RETURN d;
```

Comment:
Both the queries are not working.
Second one is a C*QL query. First one does not seem a valid query.

Reply:
A SQL query should look like `select _key, value from <collection> where groupID = <groupID"`

Reply:
I got the queries from the QA handoff video, FYI.

Reply:
For C8QL queries, I generally tag them as SQL for markdown rendering, because it doesn't know what to do with C8QL. Can you correct them for C8QL?

Reply:
Updated